docs/_posts/2018-06-26-jekyll-sponsoring.markdown

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Jekyll and Sponsoring'
+date: 2018-06-26 16:00:00 +0200
+author: oe
+categories: [community]
+---
+
+_(TL;DR: We're open for sponsorships over our [OpenCollective page](https://opencollective.com/jekyll))_
+
+Hi Jekyllers,
+
+As you may know, Jekyll is a completely free and open source project. We offer our
+software and its related plugins and documentation at no cost because we believe
+that good software should not cost anything. We're not planning on changing that,
+but today I want to talk about a different monetary aspect of open source.
+
+Open source developers being paid for the work they do is a rare sight. Most open source
+software is effectively the result of hundreds and thousands of hours of free labor provided
+by individuals who are passionate enough to work outside of their day job to create
+software that, ironically, is being used by almost every company that offers
+digital services. It's a problem that has gotten more attention in recent years, with
+the open source community becoming more diverse and more and more companies actively
+investing in providing monetary support for open source developers.
+
+Jekyll has always been a product of volunteers. Rarely has someone been paid to implement
+a certain plugin or feature. Today, we're excited to announce that we will finally
+be able to fund our contributors! __We are opening an OpenCollective to receive
+individual and corporate sponsorships__. This is not unheard of, [Hugo](http://gohugo.io)
+is also funded by sponsorships, as are many other similar projects.
+
+OpenCollective is a service that makes it easy for open source projects to receive funding
+from individuals and companies alike. It's specifically designed for open source and
+many other projects already use it for funding.
+
+With that, we would like to announce our very first sponsor: [__Forestry.io__](https://forestry.io)! Forestry is
+a CMS that integrates with your Jekyll sites and lets you update content using a beautiful
+interface, and then automatically commits it back to your GitHub repository. We're excited to have
+them on board on a new, exciting step of our journey.
+
+Will anything change for Jekyll users? The answer is no - this step does not impact the Jekyll software
+in any aspect. In fact, you might see positive changes, such as more features and better
+performance. Surprisingly, that's what happens when you properly fund people for their work!
+
+If you have been a long time user for Jekyll and would like to give something back to the project,
+you can consider a small monthly donation to our [OpenCollective page](http://opencollective.com/jekyll).
+If your company heavily relies on Jekyll, consider sponsoring us! Contact me at
```

Comment:
`please consider sponsoring us` ?

Reply:
well we don't want to sound like we're begging for it :wink:

Reply:
`do consider sponsoring us!` then.. :stuck_out_tongue: